.github/workflows/ci.yml

```diff
@@ -39,6 +39,8 @@ jobs:
         with:
           components: clippy
       - run: cargo clippy --all-features --all-targets
+      - run: cargo clippy --no-default-features --features ring --all-targets
+      - run: cargo clippy --no-default-features --features aws_lc_rs --all-targets
```

Comment:
Should I add these combinations to `cargo test` as well?

Reply:
I went ahead and parallelized the Clippy runs as well.
The `cargo test`s seem to not be straightforward to add to a matrix, so I refrained from touching those.